scripts/auto-rebase/rebase_job_entrypoint.sh

```diff
@@ -41,14 +61,24 @@ ARM64_RELEASE=${pullspec_release_arm64} \
 #  For example, ocp-release at 4.13 will more often than not
 #  correspond to 4.12 LVMS, until the official 4.13 release when both
 #  components will be 4.13.
-release_lvms="v4.15.0"
+release_lvms="v4.17.0-18"
 
 # Since LVMS is not part of the release payload, it is not kept in
 # CI. Use the latest z-stream that coincides with the release
 # payload's X.Y version
 pullspec_release_lvms="registry.redhat.io/lvms4/lvms-operator-bundle:${release_lvms}"
+# A unreleased candidate doesnt exist in the official registry, so fallback to the lvms_dev namespace, which contains
+# the latest lvms release candidate replicated from CPaaS into quay
+pullspec_release_lvms_fallback="quay.io/lvms_dev/lvms4-lvms-operator-bundle:${release_lvms}"
+
+# You can also pass the version as a command-line argument
```

Comment:
Is this comment from a previous iteration? The script does not parse command line args.